redisinsight/api/src/modules/rdi/rdi.service.ts

```diff
@@ -38,6 +38,12 @@ export class RdiService {
     return Object.keys(omitBy(dto, isUndefined)).some((field) => this.connectionFields.includes(field));
   }
 
+  private static async getRdiVersion(client: RdiClient): Promise<string> {
+    const pipelineStatus = await client.getPipelineStatus();
+    const version = pipelineStatus.components.processor.version;
```

Comment:
And I was wondering if there's a scenario where this won't be here. For example legacy version of RDI, etc... Do we have default value for here like `N/A` @ViktarStarastsenka

Reply:
@pawelangelow , let's display "-" if no value is received. Just in case.